src/main/resources/com/microsoft/jenkins/containeragents/aci/dns/AciDnsConfig/config.jelly

```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+
+    <f:entry title="Name of the DNS servers">
```

Comment:
do we need a title here?

and is the entry needed?

```suggestion
    <f:entry>
```

Reply:
I use a title because the template has more fields for the dnsConfig, but I need currently only one field.

```json
    "dnsConfig": {
      "nameServers": [ "string" ],
      "options": "string",
      "searchDomains": "string"
    },
```

So we would be upward compatible, when someone want to extend it.

Reply:
I think just adjust when needed, it looks weird atm to me